LiteEditor/mainbook.cpp

```diff
@@ -612,11 +612,12 @@ LEditor* MainBook::OpenFile(const wxString& file_name, const wxString& projectNa
             AddPage(editor, fileName.GetFullName(), tooltip.IsEmpty() ? fileName.GetFullPath() : tooltip, bmp);
         }
         editor->SetSyntaxHighlight();
+        ManagerST::Get()->GetBreakpointsMgr()->RefreshBreakpointsForEditor(editor);

```

Comment:
Dunno if this is the right thing to do or the right place to do it, but seems to work!